platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -396,6 +398,8 @@
 		.de("Initialisiert");
 	I18n1 INVALID_IP_ADDRESSES_ARG1 = new I18n1("Invalid IP addresses: %s")//
 		.de("Ungültige IP-Addressen: %s");
+	I18n0 IS_NOT_QUEUED = new I18n0("Is not queued")//
```

Comment:
The leading `IS_` / `"Is "` should be removed. Otherwise, we'd need to prefix all predicates with "is". Also, in the German translation, there is no "ist".